Translators/WZDx/main.py

```diff
@@ -75,6 +75,66 @@ def translate(wzdx_geojson):
             logging.info(f'Failed to generate TIM for feature: {feature["id"]}')
     return tims
 
+def translate(wzdx_geojson):
+    tims = {"timRwList": []}
+
+    for feature in wzdx_geojson:
+        if (len(feature["geometry"]["coordinates"]) <= 2):
+            continue
+        tim_body = {}
+        tim_body["direction"] = "I" if feature["properties"]["core_details"]["direction"].lower() in ["northbound", "eastbound"] else "D"
+        tim_body["bearing"] = get_bearing(feature)
+        tim_body["geometry"] = get_geometry(feature["geometry"]["coordinates"])
+        tim_body["route"] = feature["properties"]["core_details"]["road_names"][0].replace("_", "-")

```

Comment:
(non-blocking) Using a model class for this 'feature' object would make the code much easier to digest.

For example, this line could be rewritten as the following:
```
tim_body["route"] = feature.get_first_road_name()
```